Core/MSBuild/MSBuild.cs

```diff
@@ -46,22 +46,21 @@ public void CallMSBuild (string msBuildPath, string arguments)
 
     var msBuildStartInfo = new ProcessStartInfo(msBuildPath, arguments);
     msBuildStartInfo.UseShellExecute = false;
-    msBuildStartInfo.RedirectStandardError = true;
     msBuildStartInfo.RedirectStandardOutput = true;
     msBuildStartInfo.WorkingDirectory = Environment.CurrentDirectory;
 
     var msBuildProcess = Process.Start(msBuildStartInfo);
+
+    var msBuildOutput = msBuildProcess!.StandardOutput.ReadToEnd();
+    
     msBuildProcess!.WaitForExit(c_msBuildProcessTimeout);
 
     if (msBuildProcess.ExitCode != 0)
     {
-      var errorMessage = msBuildProcess.StandardError.ReadToEnd();
-      var errorOutput = msBuildProcess.StandardOutput.ReadToEnd();
-      var message = $"MSBuild '{arguments}' failed with Error: '{errorOutput}'.";
+      var message = $"MSBuild '{arguments}' failed with Error: '{msBuildOutput}'.";
       throw new Exception(message);
```

Comment:
Do we already have a cleanup task to change Exception to e.g. InvalidOperationException?